charts/v0.3.0/gmsa/templates/_helpers.tpl

```diff
@@ -0,0 +1,7 @@
+# Rancher
```

Comment:
is this a generic helper or rancher specific? We shouldn't bring in provider specific logic to the repo.